plugin.video.itvrs/resources/lib/itvrs.py

```diff
@@ -0,0 +1,86 @@
+
+# -*- coding: utf-8 -*-
+#------------------------------------------------------------
+# License: GPL (http://www.gnu.org/licenses/gpl-3.0.html)
+# Based on code from youtube addon
+#------------------------------------------------------------
+
+import os
+import sys
+import plugintools
+import xbmc,xbmcaddon
+from addon.common.addon import Addon
+
+addonID = 'plugin.video.itvrs'
+addon = Addon(addonID, sys.argv)
+local = xbmcaddon.Addon()
+icon = local.getAddonInfo('icon')
+
+YOUTUBE_CHANNEL_ID_1 = "UCB2U9Zvptf24QzBnNKHUtsw"
+
+# Entry point
+def run():
+    plugintools.log("itvrs.run")
+    
+    # Get params
+    params = plugintools.get_params()
+    
+  #  if params.get("action") is None:
```

Comment:
I think you commented a bit more than necessary. My comment was because of the else statement

Reply:
@enen92, I think I got it now.